### PR TITLE
[IIIF-803] Add procedure for installing Festerize with Pip

### PR DESCRIPTION
**Pull request by markmatney**
I will ask DLP to review.

### CODE REVIEW

File: src/main/scripts/festerize/README.md [hunk 1]
```diff
@@ -4,30 +4,11 @@ Uploads CSV files to the Fester IIIF manifest service.
...
-If you want to install it in a place that's available to all users on a system (and you have sudo privileges) you can use:
-
-    sudo ./setup.py
-
-### Installing on a Mac
-
-To install on a Mac that's installed Python 3 with Homebrew, type:
-
-    python3 setup.py install
+    bash <(curl -sSL https://raw.githubusercontent.com/UCLALibrary/fester/master/install.sh)
```
Comment (ksclarke) [inline src/main/scripts/festerize/README.md]: Is this path right? It should be the `src/main/scripts/festerize` path right?

File: src/main/scripts/festerize/install.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    if ! [ -x "$(command -v ${PIP})" ]; then
+        echo "${PIP} not installed" >&2
+        PIP=
+    else
+        break
+    fi
+done
+
+if [ -z ${PIP} ]; then
+    echo "A version of Pip compatible with Python 3 is required to install Festerize."
```
Comment (ksclarke) [inline src/main/scripts/festerize/install.sh:14]: Does the above actually check that `pip` is v3? Are there systems for which `pip` might be pointed to a 2.x version?

File: src/main/scripts/festerize/README.md [hunk 3]
```diff
@@ -4,30 +4,11 @@ Uploads CSV files to the Fester IIIF manifest service.
 
 ## Installation
 
-You need to have Python 3 installed to use Festerize. This README will describe the simplest way to get started; if you're more experienced and would like to use a virtualized Python environment, feel free to do that.
+First, ensure that you have Python 3 and Pip installed on your system.
```
Comment (ksclarke) [inline src/main/scripts/festerize/README.md:7]: And curl and bash, right?

File: src/main/scripts/festerize/README.md (hunk 1 above)
Comment (markmatney) [reply src/main/scripts/festerize/README.md]: yeah I botched it, just pushed another commit that fixes this

File: src/main/scripts/festerize/README.md (hunk 3 above)
Comment (markmatney) [reply src/main/scripts/festerize/README.md:7]: ah, true

File: src/main/scripts/festerize/install.sh (hunk 2 above)
Comment (ksclarke) [reply src/main/scripts/festerize/install.sh:14]: It could parse `pip -V` output perhaps?

Comment (markmatney) [reply src/main/scripts/festerize/install.sh:17]: Yeah, I think that will work. I verified that it works on a Mac too. Just pushed a new commit.

**Pull request merged**